<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="AlgorithmForm" name="widgetAlgorithm" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DicomDataForm" name="widgetDataInfo" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="FourPanelForm" name="fourPanelWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="prueba">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionSpanish"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menu_Language"/>
    <addaction name="action_Histogram"/>
    <addaction name="action_Algorithm"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Transversal_view"/>
    <addaction name="actionCoronal_view"/>
    <addaction name="action_Sagital_view"/>
    <addaction name="action_Volume_View"/>
    <addaction name="action_All_views"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Histogram"/>
   <addaction name="action_Transversal_view"/>
   <addaction name="actionCoronal_view"/>
   <addaction name="action_Sagital_view"/>
   <addaction name="action_Volume_View"/>
   <addaction name="action_All_views"/>
   <addaction name="action_Algorithm"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>&amp;Spanish</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Histogram">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/Icons/histograma.png</normaloff>:/Icons/Icons/histograma.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Histogram</string>
   </property>
  </action>
  <action name="action_Transversal_view">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/Icons/VistaTransversal.png</normaloff>:/Icons/Icons/VistaTransversal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Transversal view</string>
   </property>
   <property name="toolTip">
    <string>Transversal view</string>
   </property>
  </action>
  <action name="actionCoronal_view">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/Icons/VistaCoronal.png</normaloff>:/Icons/Icons/VistaCoronal.png</iconset>
   </property>
   <property name="text">
    <string>Coronal view</string>
   </property>
  </action>
  <action name="action_Sagital_view">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/Icons/VistaSagital.png</normaloff>:/Icons/Icons/VistaSagital.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Sagital view</string>
   </property>
  </action>
  <action name="action_All_views">
   <property name="text">
    <string>&amp;All views</string>
   </property>
  </action>
  <action name="action_Volume_View">
   <property name="text">
    <string>&amp;Volume View</string>
   </property>
  </action>
  <action name="action_Algorithm">
   <property name="text">
    <string>&amp;Algorithm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DicomDataForm</class>
   <extends>QWidget</extends>
   <header location="global">dicomdataform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FourPanelForm</class>
   <extends>QWidget</extends>
   <header location="global">fourpanelform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlgorithmForm</class>
   <extends>QWidget</extends>
   <header location="global">algorithmform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
